core/src/main/java/com/yahoo/ycsb/ByteIterator.java

```diff
@@ -56,16 +56,6 @@ public Byte next() {
 
   public abstract byte nextByte();
 
-  /** @return byte offset immediately after the last valid byte */
-  public int nextBuf(byte[] buf, int bufOff) {
```

Comment:
Keep these around since a private DB implementation may rely on it.

Reply:
Done.